lib/ui/view/channel/pinned_messages/pinned_messages_view.dart

```diff
@@ -24,7 +24,8 @@ class PinnedMessagesView extends StatelessWidget {
             leading: Icons.arrow_back_ios,
             orgTitle: Text(
               PinnedMessages,
-              style: AppTextStyles.heading7,
+              style:
+                  AppTextStyles.heading4.copyWith(color: AppColors.blackColor),
```

Comment:
You added copyWith - It will affect the theme. Besides heading 4 has black in it